### PR TITLE
OSDOCS-1350 Separate Builds command and output blocks

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-1350

Note - went ahead with some additional formatting fixes while going through the files.

### CODE REVIEW

File: modules/builds-basic-start-source.adoc [hunk 1]
```diff
@@ -8,45 +8,36 @@ Rather than relying on a Git source pull
...
 ====
 
 .Procedure
 
-For example, the following command sends the contents of a local Git repository
-as an archive from the tag `v2` and starts a build:
+Start a build from a source.
 
+The following command sends the contents of a local Git repository as an archive from the tag `v2` and starts a build:
+
```
Comment (mburke5678) [inline modules/builds-basic-start-source.adoc:39]: Do we need both of these lines? Something like: 
_Start a build from a source using the following command to send the contents of a local Git repository as an archive from the tag `v2`:_

File: modules/builds-use-custom-builder-image.adoc [hunk 2]
```diff
@@ -71,13 +68,9 @@ $ oc create -f imagestream.yaml
 
 . Run your custom build:
 +
+[source,terminal]
```
Comment (mburke5678) [inline modules/builds-use-custom-builder-image.adoc:71]: Did you miss the oc commands on lines 66 and 49?

File: modules/builds-setting-triggers-manually.adoc [hunk 3]
```diff
@@ -5,34 +5,39 @@
...
 ====
 
-For more information, consult the help documentation with `oc set triggers
---help`
+For more information, consult the help documentation with by running:
+
+[source,terminal]
+----
+ `oc set triggers --help`
+----
```
Comment (mburke5678) [inline modules/builds-setting-triggers-manually.adoc:43]: Should you remove the ticks here and add the $ prompt? I know what is being done here, but it looks weird. 
Definitely remove the ticks as they show up in the output.

File: modules/builds-troubleshooting-access-resources.adoc [hunk 4]
```diff
@@ -10,6 +10,8 @@ If your request for access to resources is denied:
 Issue::
 A build fails with:
 
+.Example output
```
Comment (mburke5678) [inline modules/builds-troubleshooting-access-resources.adoc:13]: Do we need the .Example output here? Is the output an actual string, not an example??

File: modules/builds-troubleshooting-service-certificate-generation.adoc [hunk 5]
```diff
@@ -8,24 +8,29 @@
...
 secret/ssl-key references serviceUID 62ad25ca-d703-11e6-9d6f-0e9c0057b608, which does not match 77b6dd80-d716-11e6-9d6f-0e9c0057b60
 ----
 
 Resolution::
-The service that generated the certificate no longer exists, or has a different
-`serviceUID`. You must force certificates regeneration by removing the old
-secret, and clearing the following annotations on the service
-`service.alpha.openshift.io/serving-cert-generation-error`,
-`service.alpha.openshift.io/serving-cert-generation-error-num`:
+The service that generated the certificate no longer exists, or has a different `serviceUID`. You must force certificates regeneration by removing the old secret, and clearing the following annotations on the service `service.alpha.openshift.io/serving-cert-generation-error`, `service.alpha.openshift.io/serving-cert-generation-error-num`:
```
Comment (mburke5678) [inline modules/builds-troubleshooting-service-certificate-generation.adoc:20]: This sentence doesn't make a whole lotta sense to me!

**Comment by mburke5678** [comment]
Did you miss `configmap-adding-ca.adoc`? There are a couple of `oc` commands that don't look marked up.

**Comment by mburke5678** [comment]
@bmcelvee I reviewed the code block changes and obvious text changes. I didn't review all of the paragraphs where you removed carriage returns. I left a few comments. Otherwise LGTM

File: modules/builds-basic-start-source.adoc (hunk 1 above)
Comment (bmcelvee) [reply modules/builds-basic-start-source.adoc:37]: Fixed, thanks!

File: modules/builds-use-custom-builder-image.adoc (hunk 2 above)
Comment (bmcelvee) [reply modules/builds-use-custom-builder-image.adoc:73]: Yep, fixed. Thanks!

File: modules/builds-setting-triggers-manually.adoc (hunk 3 above)
Comment (bmcelvee) [reply modules/builds-setting-triggers-manually.adoc:43]: Thank you - updated.

File: modules/builds-troubleshooting-access-resources.adoc (hunk 4 above)
Comment (bmcelvee) [reply modules/builds-troubleshooting-access-resources.adoc]: Fixed - thanks!

File: modules/builds-troubleshooting-service-certificate-generation.adoc (hunk 5 above)
Comment (bmcelvee) [reply modules/builds-troubleshooting-service-certificate-generation.adoc]: Fair, edited.

**Comment by bmcelvee** [comment]
Thank you so much, @mburke5678!

**Comment by bmcelvee** [comment]
Having trouble with the cherry picks - both manual and automated.

**Comment by bmcelvee** [comment]
Both cherry pick PRs are failing. Will look at this over the weekend.

**Comment by bmcelvee** [comment]
Update: cherry picks are continuing to fail because of unrelated virt cross refs.

**Pull request merged**